modules/identity-provider-oidc-CR.adoc

```diff
@@ -22,43 +22,35 @@ metadata:
 spec:
   identityProviders:
   - name: oidcidp <1>
-    challenge: false <2>
-    login: true <3>
-    mappingMethod: claim <4>
+    mappingMethod: claim <2>
     type: OpenID
     openID:
-      clientID: ... <5>
-      clientSecret: <6>
+      clientID: ... <3>
+      clientSecret: <4>
         name: idp-secret
-      claims: <7>
+      claims: <5>
         preferredUsername:
         - preferred_username
         name:
         - name
         email:
         - email
       urls:
```

Comment:
`urls` was dropped openshift/api#265 and replaced with `issuer`

Reply:
Thanks for catching that! This is now updated.

Reply:
Discussed this with @huffmanca and he will provide further edits in a follow-up PR